src/components/MenuItem.tsx

```diff
@@ -643,7 +643,8 @@ function MenuItem(
                                                     )}
                                                     {shouldShowAvatar && shouldShowSubscriptAvatar && (
                                                         <SubscriptAvatar
-                                                            backgroundColor={isHovered ? theme.hoverComponentBG : undefined}
+                                                            // eslint-disable-next-line no-nested-ternary
+                                                            backgroundColor={pressed ? theme.buttonHoveredBG : isHovered ? theme.hoverComponentBG : undefined}
```

Comment:
Let's fix this instead of silencing the lint

Reply:
@s77rt Fixed.